javascript/collapsible.js

```diff
@@ -1,90 +1,107 @@
 var tableRows = document.getElementsByTagName('tr');
 
-function toggleGroup(groupName) {
-  for (var i = 0; i < tableRows.length; i++) {
-    if (tableRows[i].title == groupName) {
-      if (tableRows[i].style.display == 'none') {
-        tableRows[i].style.display = 'table-row';
-      } else {
-        tableRows[i].style.display = 'none';
-      }
+function toggleGroup(groupName) 
+{
+
```

Comment:
Remove those newlines, things are pretty readable already like you have them now.

	function toggleGroup(groupName) 
	{

		some stuff

	}

	vs.

	function toggleGroup(groupName) 
	{
		some stuff
	}

The former is only a waste of visual space in many cases. To get that visual space, you have moved the `{` down the line already.

Reply:
I have done this change in the new commit.